Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```diff
@@ -22,207 +33,69 @@ span.bocDateValue
   min-width: 8em;
 }
 
+span.bocDateTimeValue,
+span.bocDateValue,
 span.bocDateTimeValue.readOnly,
 span.bocDateValue.readOnly
 {
-  height: inherit;
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
-}
-
-span.bocDateTimeValue.disabled,
-span.bocDateValue.disabled
-{
-}
-
-.mozilla span.bocDateInputWrapper
-{
-  right: 2px;
+  display: flex;
+  align-items: center;
```

Comment:
I think it's enough to put those two with span.bocDateTimeValue/span.BocDateValue in line 15. The `.readonly` gets all from the first class anyway.